common/eslint-js.config.mjs

```diff
@@ -0,0 +1,16 @@
+import { createRequire } from 'module';
+const require = createRequire(import.meta.url);
+
+const tsEslint = require('typescript-eslint');
+const eslint = require('@eslint/js');
+const eslintPluginPrettierRecommended = require('eslint-plugin-prettier/recommended');
```

Comment:
Same thing here.  createRequire() interesting.  A backward compatibility shim?

Reply:
The dependencies are packaged as commonjs, not ES modules.